torch_uncertainty/datamodules/cifar10.py

```diff
@@ -117,7 +142,11 @@ def setup(self, stage: Optional[str] = None) -> None:
                 transform=self.transform_train,
             )
             self.train, self.val = random_split(
-                full, [len(full) - self.val_split, self.val_split]
+                full,
+                [
+                    int(len(full) ** (1 - self.val_split)),
```

Comment:
Why the double ** here? Shouldn't it be simply * instead?
Other datamodules have the same implementation

Reply:
You're right! Changing it now